src/components/SimpleModal/SimpleModal.layouts.jsx

```diff
@@ -47,3 +58,45 @@ HeaderAndFooter.propTypes = {
   /** Custom header content */
   header: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
 }
+
+export function Confirmation({
+  heading,
+  body,
+  onConfirm = noop,
+  onCancel = noop,
+  confirmButtonText = 'Accept',
+  cancelButtonText = 'Cancel',
+  danger,
+}) {
+  return (
+    <ConfirmationWrapperUI className={classNames(!body && 'is-compact')}>
+      <ConfirmationHeadingUI>{heading}</ConfirmationHeadingUI>
+      {body && <ConfirmationBodyUI>{body}</ConfirmationBodyUI>}
+      <ConfirmationFooterUI>
+        <Button outlined theme="grey" onClick={onConfirm}>
+          {cancelButtonText}
+        </Button>
+        <Button theme={danger ? 'red' : 'blue'} onClick={onCancel}>
+          {confirmButtonText}
+        </Button>
+      </ConfirmationFooterUI>
+    </ConfirmationWrapperUI>
+  )
+}
+
+Confirmation.propTypes = {
+  /** Text of the modal heading */
+  heading: PropTypes.string.isRequired,
+  /** Text of the modal body */
+  body: PropTypes.string,
```

Comment:
Can we make the body accept a React Node instead of string? I think we may need to apply text formatting, i.e. to create emphasis.

Reply:
Of course! No problem, I added that to the heading as well 👍